.travis.yml

```diff
@@ -1,22 +1,47 @@
 language: node_js
 env:
-  - ASCIIDOCTOR_CORE_VERSION=v1.5.5
-  - ASCIIDOCTOR_CORE_VERSION=master
+- ASCIIDOCTOR_CORE_VERSION=v1.5.5
+- ASCIIDOCTOR_CORE_VERSION=master
 node_js:
-  - "4"
-  - "6"
-  - "node"
-
+- '4'
+- '6'
+- node
 jdk:
-  - oraclejdk8
-
+- oraclejdk8
 before_script:
-  - npm install
-
+- npm install
 script:
- - npm run lint
- - npm run package
- - npm run docs
- - SKIP_BUILD=1 npm run examples
-
+- npm run lint
+- npm run package
+- npm run docs
+- SKIP_BUILD=1 npm run examples
 sudo: false
+
+before_deploy:
+- npm run build
```

Comment:
No it's not necessary since `npm run dist` runs `build`: https://github.com/asciidoctor/asciidoctor.js/blob/07ce5da718e0234f83e6f16362f0fcc6735dd3c2/npm/builder.js#L159